app/helpers/application_helper.rb

```diff
@@ -1,4 +1,8 @@
 module ApplicationHelper
+  def availability_collection
+    [:flexible, :morning, :afternoon, :evening, :workday, :weekend]
```

Comment:
if we have this as a concern, why do we need it here again? for the translations?